### PR TITLE
feat(css/crowding): Make crowding information colors more accessible

### PR DESCRIPTION
**Pull request by firestack**
---

asana ticket: https://app.asana.com/0/1203014709808707/1195589034084553/f

### CODE REVIEW

**Review by KaylaBrady — APPROVED** [review]
Looks good to me! To confirm - is the fill color change in the ticket not needed?

**Comment by firestack** [comment]
> To confirm - is the fill color change in the ticket not needed?

@KaylaBrady I couldn't tell where to apply it, it seems for .com, they have a colored background for the text (`Not Crowded|Some Crowding|Crowded`) as shown by the last photo added in the original ticket 

- https://app.asana.com/0/385363666817452/1187313467059937/f
- (I dunno if this'll properly direct link to the photo) https://app.asana.com/app/asana/-/get_asset?asset_id=1189109534658732

I didn't see any place I could apply that same change in skate, like, I could try and set the background behind the crowding text in the VPP?

File: assets/css/_crowding.scss [hunk 1]
```diff
@@ -64,22 +64,6 @@ $color-crowding-icon-bg: $color-bg-medium;
 .m-crowding-diagram__status-description {
   font-weight: 500;
   text-transform: uppercase;
-
-  &--empty {
-    color: $color-crowding-empty-text;
```
Comment (KaylaBrady) [inline assets/css/_crowding.scss:69]: nitpick(non-blocking): Is the `$color-crowding-empty-text` variable still needed?

**Review by KaylaBrady — APPROVED** [review]

File: assets/css/_crowding.scss (hunk 1 above)
Comment (firestack) [reply assets/css/_crowding.scss:69]: Nice catch! seems like it's not, but I could actually leave the `--empty` selector and keep styling the empty text the same.

That's sorta underspecified in the ticket.

**Review by lemald — APPROVED** [review]
Looks good to me! 👍

**Pull request merged**